projects/ad9081/src/app_iio.c

```diff
@@ -56,6 +56,11 @@
 /************************** Functions Implementation **************************/
 /******************************************************************************/
 
+struct iio_data_buffer g_read_buff = {
+		.buff = (void *)ADC_DDR_BASEADDR,
+		.size = 0xFFFFFFFF,
```

Comment:
the `size` value seems to be propagated throughout all projects that support IIO. is it possible to add a macro for this?

Reply:
I think I can change this if needed in a other PR for both ADC and DAC. Will make more sense together.